packages/flex-plugin-scripts/package.json

```diff
@@ -41,20 +41,23 @@
   },
   "dependencies": {
     "@jest/types": "^26.6.2",
-    "@types/enzyme": "^3.10.6",
-    "@types/jest": "26.0.14",
-    "@types/node": "^12.7.12",
+    "@types/enzyme": "^3.10.8",
+    "@types/jest": "26.0.20",
+    "@types/node": "^14.14.31",
     "@types/react": "^16.8.16",
     "@types/react-dom": "^16.8.4",
     "@types/react-redux": "5.0.21",
+    "emotion": "9.2.12",
     "esm": "^3.2.25",
     "flex-dev-utils": "^4.4.0-beta.0",
     "flex-plugin": "^4.4.0-beta.0",
     "flex-plugin-test": "^4.4.0-beta.0",
     "flex-plugin-webpack": "^4.4.0-beta.0",
-    "form-data": "^2.5.1",
-    "jest": "26.5.3",
-    "querystring": "^0.2.0",
+    "form-data": "^4.0.0",
+    "jest": "26.6.3",
+    "prop-types": "^15.7.2",
+    "querystring": "^0.2.1",
+    "react": "^16",
```

Comment:
we can do this now? 😯

Reply:
npm 7 ;) It will install the versino we have provided in the package.json. And it will stop all the warnings about peer dependency not installed.